### ISSUE TITLE
[HC] The icon of "Browse..." in PropertyGrid control is unclear when non-selecting it

### ISSUE THREAD

**Issue by MelonWang1** [dotnet/winforms#7721]
### .NET version

.NET 8.0.100-alpha.1.22451.15

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, it is not a regression issue

### Issue description

**Current Behavior:**
The icon of "Browse..." is unclear when non-selecting it. 
HC Aquatic theme:
<img width="217" alt="11-Aquatic" src="https://user-images.githubusercontent.com/94418985/188046392-229f624d-29e6-42be-81f1-738a4c702363.png">
HC Dusk theme:
<img width="221" alt="11-Dusk" src="https://user-images.githubusercontent.com/94418985/188046443-29f0010f-bb14-4716-93b1-dc7d7a3f77f7.png">
HC Night Sky theme:
<img width="218" alt="11-Night Sky" src="https://user-images.githubusercontent.com/94418985/188046483-ab5e7692-d9fa-458e-bca7-e2346597ac00.png">
**Expected Behavior:**
The icon of "Browse..." should be clear when non-selecting it. 
HC Aquatic theme:
<img width="372" alt="22-Aquatic" src="https://user-images.githubusercontent.com/94418985/188046639-cfb96280-6bab-48c9-ad7a-c85e79e1c471.png">
HC Dusk theme:
<img width="367" alt="22-Dusk" src="https://user-images.githubusercontent.com/94418985/188046669-9d685632-a158-4ba9-aadc-809ce80bf4ea.png">
HC Night Sky theme:
<img width="352" alt="22-Night sky" src="https://user-images.githubusercontent.com/94418985/188046698-a4813b51-dd2c-4482-bec5-d1dd8cbd90df.png">

### Steps to reproduce

1. Turn on High Contrast theme in Windows settings 
2. Create a WinForms .NET core project  
3. Add a PropertyGrid control and resize it 
4. Choose it "SelectedObject"properties to "Form1" 
5. Ctrl+Shift+A add a Application Manifest File and uncomment windows 10
6. Build and run 
7. Observe the icon of "Browse..." 

More Info:
1. This issue cannot repro on win11 HC Desert theme. 
2. This issue is repro on win10 machine but cannot repro on win10 HC White theme. 
3. This issue cannot repro on framework project.

**Comment by merriemcgaw** [comment]
@dmitrii-drobotov / @vladimir-krestov - what is the difference between the glyphs in .NET and .NET Framework? Do we do anything different on our end?

**Comment by dmitrii-drobotov** [comment]
> @dmitrii-drobotov / @vladimir-krestov - what is the difference between the glyphs in .NET and .NET Framework? Do we do anything different on our end?

Yes, it was changed in #886. The difference is that now there is a white border around dots (left is the old icon, right is the current one):
![image](https://user-images.githubusercontent.com/102954094/189163559-1e666a6a-dd91-4e4c-88de-bbee01a9797a.png)

**Comment by dmitrii-drobotov** [comment]
@NikitaSemenovAkvelon - do you think we could reuse the same approach as in #8507 for this issue?

**Comment by NikitaSemenovAkvelon** [comment]
@dmitrii-drobotov, yeah, I guess it would work here.

**Comment by John-Qiao** [comment]
Verified this issue in the latest .NET 8.0 SDK build: 8.0.100-preview.7.23323.19, it was fixed that icon of "Browse..." is clear now in both Win 10 and Win 11 OS High Contrast themes.

Win 10:
![W10-testresult](https://github.com/dotnet/winforms/assets/45864985/8f9dc9b6-c0c7-49db-a6df-5df232a8e8a2)

Win 11:
![W11-testresult](https://github.com/dotnet/winforms/assets/45864985/cb4c4b0d-257e-4e87-b5a4-f5738f34afa2)

**Comment by MelonWang1** [comment]
Verified this on .NET 8.0 TP build in Win11 22H2 & Win10 22H2 OS: 8.0.100-preview.6.23328.9, issue was fixed, result is same as above test result.
